trulens_eval/trulens_eval/feedback/v2/feedback.py

```diff
@@ -220,61 +285,95 @@ class ContextRelevance(Relevance, WithPrompt):
 
         - RELEVANCE score should increase as the CONTEXTS provides RELEVANT context to more parts of the QUESTION.
 
-        - CONTEXT that is RELEVANT to some of the QUESTION should score of 2, 3 or 4. Higher score indicates more RELEVANCE.
-
-        - CONTEXT that is RELEVANT to most of the QUESTION should get a score of 5, 6, 7 or 8. Higher score indicates more RELEVANCE.
+        - Never elaborate.
+        """
+        )
+    )
 
-        - CONTEXT that is RELEVANT to the entire QUESTION should get a score of 9 or 10. Higher score indicates more RELEVANCE.
+    @staticmethod
+    def validate_criteria_and_output_space(criteria: str, output_space: str):
+        validated = EvalCriteria(criteria=criteria, output_space=output_space)
+        return validated
 
-        - CONTEXT must be relevant and helpful for answering the entire QUESTION to get a score of 10.
+    @classmethod
+    def override_critera_and_output_space(
+        cls, criteria: str, output_space: str
+    ):
+        validated = cls.validate_criteria_and_output_space(
+            criteria, output_space
+        )
+        cls.output_space_prompt = validated.get_output_scale_prompt()
+        cls.system_prompt = PromptTemplate.from_template(
+            cls.system_prompt_template.format(
+                criteria=validated.criteria,
+                output_space_prompt=cls.output_space_prompt,
+            )
+        )
 
-        - Never elaborate."""
-    )
-    user_prompt: ClassVar[PromptTemplate] = PromptTemplate.from_template(
+    user_prompt = PromptTemplate.from_template(
         """QUESTION: {question}
-
         CONTEXT: {context}
 
-        RELEVANCE: """
+        RELEVANCE:
+        """
+    )
+    verb_confidence_prompt = PromptTemplate.from_template(
+        """Finally after generating the RELEVANCE score, provide the confidence level CONFIDENCE on a scale of 0 to 10 that your RELEVANCE scoring is accurate (i.e. how confident you are with your evaluation score). Give ONLY the confidence score, no
+        other words or explanation.\n\nFor example: CONFIDENCE: <the probability between
+        0 and 10 that your scoring is accurate, without any extra commentary whatsoever;
```

Comment:
Probability should be between 0 & 1?

Reply:
thanks for catching this. updated prompt + score normalization to use real probability